app/uk/gov/hmrc/agentsubscriptionfrontend/models/Address.scala

```diff
@@ -137,7 +136,4 @@ object Address{
 
     OFormat[Address](reads, formatAddressValue)
   }
-
-  val renderErrors: PostCodeError => String = (errors: PostCodeError) => errors.foldLeft("")(_+", "+_).substring(1)
```

Comment:
according to the renderErrors anonymous function: this is the updated, https://github.com/hmrc/agent-subscription-frontend/pull/78/commits/197c51130aac51d8e9dbef15f2fe731fa42f4be6, (commit 197c51130aac51d8e9dbef15f2fe731fa42f4be6). and all the tests are passed